starknet_py/tests/e2e/contract_interaction/interaction_test.py

```diff
@@ -219,6 +219,7 @@ async def test_general_simplified_deployment_flow(account, map_compiled_contract
 @pytest.mark.asyncio
 async def test_deploy_contract_flow(account, map_compiled_contract, map_class_hash):
     abi = create_compiled_contract(compiled_contract=map_compiled_contract).abi
+    assert abi is not None
```

Comment:
Casts don't generate errors I think. You can always do a explicit check like
```python
if abi is None:
  raise ValueError(...)
```

Reply:
Or maybe `create_contract_class` could return a subclass of `ContractClass` that has an ABI not None always?

Reply:
I don't really like any of the solutions. The first one is basically the same as current asserts and would probably lower the coverage (as it would be a pain to test all such cases where it should not even be possible to have such condition) and the second adds extra complexity